applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -17,6 +17,8 @@
 #include "includes/define.h"
 #include "custom_python/add_custom_utilities_to_python.h"
 
+#include "custom_processes/output_quadrature_domain_process.h"
```

Comment:
I would add this to a new file called add_custom_processes_to_python and add this there. It is done like this in most of the other apps.

Reply:
as you wish haha ;)